lib/workers/repository/process/limits.ts

```diff
@@ -1,141 +1,79 @@
 import { DateTime } from 'luxon';
 import type { RenovateConfig } from '../../../config/types';
 import { logger } from '../../../logger';
-import type { Pr } from '../../../modules/platform';
 import { platform } from '../../../modules/platform';
 import { scm } from '../../../modules/platform/scm';
 import { ExternalHostError } from '../../../types/errors/external-host-error';
 import type { BranchConfig } from '../../types';
 
-export async function getPrHourlyRemaining(
+export async function getPrHourlyCount(
   config: RenovateConfig,
 ): Promise<number> {
-  if (config.prHourlyLimit) {
-    try {
-      logger.debug('Calculating hourly PRs remaining');
-      const prList = await platform.getPrList();
-      const currentHourStart = DateTime.local().startOf('hour');
-      logger.debug(`currentHourStart=${String(currentHourStart)}`);
-      const soFarThisHour = prList.filter(
-        (pr) =>
-          pr.sourceBranch !== config.onboardingBranch &&
-          pr.sourceBranch.startsWith(config.branchPrefix!) &&
-          DateTime.fromISO(pr.createdAt!) > currentHourStart,
-      );
-      const prsRemaining = Math.max(
-        0,
-        config.prHourlyLimit - soFarThisHour.length,
-      );
-      logger.debug(`PR hourly limit remaining: ${prsRemaining}`);
-      return prsRemaining;
-    } catch (err) {
-      // istanbul ignore if
-      if (err instanceof ExternalHostError) {
-        throw err;
-      }
-      logger.error({ err }, 'Error checking PRs created per hour');
-      return config.prHourlyLimit;
+  try {
+    const prList = await platform.getPrList();
+    const currentHourStart = DateTime.local().startOf('hour');
```

Comment:
I think we should rather use UTC time here, so we are using internally UTC and user facing local time ( e.g. schedules )